### DISCUSSION TITLE
Automatic CPU core count

### DISCUSSION THREAD

**Discussion by netmindz** [notAperson535/OneClick-macOS-Simple-KVM#73] [Q&A]
Is there a reason why you use a hard-coded number of cores and threads rather than grabbing the info from /proc/cpuinfo ?

As core counts rise, I would have thought there is a growing number of people with more than 4 threads on their host machine

**Comment by notAperson535** [comment]
The core count is for the allocation to the vm, and you wouldn't want all of your cores to go to the vm

**Comment by netmindz** [comment]
They are not allocated exclusively for the VMs use and automatic does mean 100% of the cores either ;)

**Reply by notAperson535** [reply]
I guess I just never thought of it lol

**Comment by netmindz** [comment]
Cool, just wondered if Apple only expected certain configurations
